MvvmCross/Mac/Mac/Views/Presenters/Attributes/MvxWindowPresentationAttribute.cs

```diff
@@ -6,18 +6,50 @@ namespace MvvmCross.Mac.Views.Presenters.Attributes
 {
     public class MvxWindowPresentationAttribute : MvxBasePresentationAttribute
     {
-        public float PositionX { get; set; } = 200;
+        public static float DefaultPositionX = 200;
```

Comment:
Did you make those properties static so you can set default values from your own code?

Reply:
No I wasn't that smart. Just didn't think that we should keep making the same DefaultPositionX for each class. But now that you mention it, good idea!

Reply:
Probably initialpositionx should be const static though

Reply:
Not possible?

![image](https://user-images.githubusercontent.com/157766/30513298-e48f3b3a-9ab5-11e7-8948-15edc667e916.png)

Reply:
Just leave it static. After the docs are updated i'm all good to merge.

Reply:
I changed the initials to public float {get;}